packages/babel-plugin-polyfill-corejs2/src/index.ts

```diff
@@ -11,27 +9,25 @@ import addPlatformSpecificPolyfills from "./add-platform-specific-polyfills";
 import { hasMinVersion } from "./helpers";
 
 import defineProvider from "@babel/helper-define-polyfill-provider";
-import * as babel from "@babel/core";
-const { types: t } = babel.default || babel;
+import { types as t } from "@babel/core";
 
 const presetEnvCompat = "#__secret_key__@babel/preset-env__compatibility";
 const runtimeCompat = "#__secret_key__@babel/runtime__compatibility";
 
-// $FlowIgnore
 const has = Function.call.bind(Object.hasOwnProperty);
 
-type Options = {|
+type Options = {
   "#__secret_key__@babel/preset-env__compatibility": void | {
-    entryInjectRegenerator: boolean,
-  },
+    entryInjectRegenerator: boolean;
+  };
   "#__secret_key__@babel/runtime__compatibility": void | {
-    useBabelRuntime: string,
-    runtimeVersion: string,
-    ext: string,
-  },
-|};
+    useBabelRuntime: string;
+    runtimeVersion: string;
+    ext: string;
+  };
+};
 
-export default defineProvider<Options>(function(
+export default defineProvider<Options>(function (
   api,
   {
     [presetEnvCompat]: { entryInjectRegenerator } = {},
```

Comment:
this line and several similar lines give the type error: `error TS2525: Initializer provides no value for this binding element and the binding element has no default value`

adding a default value to each key fixes this, but I'm not sure if that's entirely the proper solution

Reply:
This error is actually correct, the default value should have been
```js
[presetEnvCompat]: { entryInjectRegenerator } = { entryInjectRegenerator: false },
```